src/lib/TermBuilder.ml

```diff
@@ -217,8 +217,19 @@ struct
 
   exception Todo
 
-  let coe_path ~fam_line ~bdry_line ~r ~s ~bdy =
-    raise Todo
+  let coe_path ~(fam_line : S.t m) (* A : I -> U *)
+      ~(bdry_line : S.t m)         (* \y.Path fam bdry ??*)
```

Comment:
Ahh so that's how you do it! Sometimes OCaml is such a mystery.

Reply:
yeah if you parenthesize to include the `~` you get a bizarre error about "operator expected"

Reply:
😎